transitmatters/new-train-tracker
# feature: Add "last seen" info for lines with no trains

So thanks to @mathcolo and his work in #34, I had a good base to work from in implementing "last seen" information for lines that currently have no new trains running. It's not fancy, but it looks like this:

<img src="https://user-images.githubusercontent.com/2208769/99326220-e1bc4a80-2845-11eb-8a8a-e64337ee9f2d.png" height="600px">

To fully configure this in production, we'll need to add

```
npm run create-history-db
```
to our deploy process, and then
```
npm run update-history-db
```
to our crontab or something to run periodically.

 _Test plan:_
- Run `npm run create-history-db`
- Update `npm run update-history-db` so that the update function is called with `test_mode=True` and then run `npm run update-history-db` to populate the database with some info, even if there are currently no new trains
- Tweak `initial_request_data()` to pass `test_mode=True` to `get_recent_sightings_for_lines`
- Open the train tracker and verify that for lines without new trains, you see a message about when a train (not actually new in this case) was actually seen on this line.
